src/bin/cli/subcommands/mod.rs

```diff
@@ -360,15 +359,15 @@ pub fn invoice(m: &ArgMatches<'_>) -> Result<(), Error> {
 
     let (search_terms, dir) = matches_to_search(m);
 
-    field = "INVOICE-NUMBER".to_owned();
+    let field = "INVOICE-NUMBER";
     actions::with_projects(dir, &search_terms, |project| {
-        if !project.empty_fields().contains(&field) {
+        if !project.empty_fields().iter().any(|s| s == field) {
```

Comment:
actually `contains` was fine, it can actually be optimized better in certain circumstances [because it get's specialized](https://doc.rust-lang.org/stable/src/core/slice/mod.rs.html#1282-1286)

Reply:
Then `field` would have to be owned: https://doc.rust-lang.org/std/vec/struct.Vec.html#method.contains

Reply:
interesting - well, no need to over optimize anyway :D